src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+import javafx.application.Platform;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Storage storage;
+    private TaskList tasks;
+    private Gui ui;
+
+    /**
+     * Constructor for the Duke Class.
+     */
+    public Duke() {
+        String filepath = "data/duke.txt";
+        ui = new Gui();
+        //ui.printStartUp();
```

Comment:
don't retain commented out code; you can retrieve it from VCS if you need it at a later stage